src/collective/listingviews/browser/views/base.py

```diff
@@ -54,7 +55,10 @@ def __init__(self, context, request):
         self.filters = dict(
             localshort = lambda item, value: plone_util.toLocalizedTime(value, long_format=0),
             locallong = lambda item, value: plone_util.toLocalizedTime(value, long_format=1),
-            tolink = lambda item, value: '<a href="%s">%s</a>'%(item.getURL(), value)
+            tolink = lambda item, value: '<a href="%s">%s</a>'%(item.getURL(), value),
+            tagImage = lambda item, value: '<img src="%s" />' %  getImageUrl(value),
+            tagMini=lambda item, value: '<img src="%s/mini" />' % getImageUrl(value),
+            tagLarge=lambda item, value: '<img src="%s/large" />' % getImageUrl(value)
```

Comment:
what about accessibiluty?

Reply:
look at the built in plone template and make similar to that.

Reply:
Are you sure those 2 represent the most useful images sizes? Maybe its safer to list all inclucing original?

Reply:
I think we make all items linked by using 2 filters. lead_image:tagMini:tolink

Reply:
change the :tagMini to :tag_mini. We don't use Camel case anywhere else